<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosHoldAssessWindow</class>
 <widget class="QMainWindow" name="PosHoldAssessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>451</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Cascadia Mono Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Position Hold Assessment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="font">
         <font>
          <family>Cascadia Mono Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);</string>
        </property>
        <property name="text">
         <string>PLUTO ROM Assessment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTrialRun">
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Cascadia Mono Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);</string>
        </property>
        <property name="text">
         <string>Trial Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="hocGraph">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>375</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblStatus">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
